File: online-judge-solutions/Codeforces/202A.py
#https://codeforces.com/problemset/problem/202/A
#Lang		:	Python 3.8
#Time		:	92 ms
#Memory         :	0 KB
s,m=input(),{}
for i in range(len(s)):m[s[i]]=m.get(s[i],0)+1
m=sorted(m.items())
c,n=list(m)[-1][0],list(m)[-1][1]
for i in range(n):print(c,end='')

File: online-judge-solutions/Codeforces/202A.cpp
//https://codeforces.com/problemset/problem/202/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
#include<map>
using namespace std;
int main(){
	string s;
	while(cin>>s){
			map<char,int> m;
			for(char x:s) m[x]++;
			for(int i=0;i<m.rbegin()->second;i++) cout<<m.rbegin()->first;
			cout<<endl;
	}
	return 0;
}
